nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -23,8 +23,8 @@ The following settings are available for changing the behavior of saga persisten
  * `ConnectionString`: Sets the connectionstring for the storage account to be used for storing saga information.
   * NServiceBus.Azure defaults to `UseDevelopmentStorage=true`.
   * NServiceBus.Persistence.AzureStorage Version 1 defaults to `null`.
- * `CreateSchema`: Instructs the persister to create the table automatically.
-  * defaults to `true`.
+ * `CreateSchema`: Instructs the persister to create the table automatically. Defaults to `true`.
+ * `AssumeSecondaryIndicesExist`: (Added in Version 1.4) By default every new saga scans for a secondary index. To avoid unnecessary performance tax, this option allows to skip the scans assuming all saga instances are in valid form and have secondary indices. 
```

Comment:
One could argue that this should be added using a versioned partial, but I prefer to see it as an encouragement to migrate to 1.4 or use the latest one version.